src/modules/feature-modules/innovator/pages/first-time-signin/first-time-signin.config.ts

```diff
@@ -139,7 +139,8 @@ function runtimeRules(steps: FormEngineModel[], data: StepPayloadType, currentSt
         dataType: 'checkbox-array',
         label: 'Finally, choose your data sharing preferences',
         validations: { isRequired: [true, 'Choose at least one organisation'] },
-        items: organisationSharesItems
+        items: organisationSharesItems,
+        description: '<p class="nhsuk-u-margin-0"><a href="/about-the-service/who-we-are" target="_blank" rel="noopener noreferrer">What does each organisation do? (opens in a new window) </a></p>'
```

Comment:
Is the <p> tag really needed? What's the added benefit?

Also, I believe that line 40 on file first-time-signin.component.ts can be removed so it does not confuse things in the future? Does this make sense?